tests/integration/logging.tcl

```diff
@@ -23,6 +25,12 @@ if {$backtrace_supported} {
         test "Server is able to generate a stack trace on selected systems" {
             r config set watchdog-period 200
             r debug sleep 1
+            if {$threads_mngr_supported} {
+                assert_equal [count_log_message 0 "failed to open /proc/"] 0
+                assert_equal [count_log_message 0 "failed to find SigBlk or/and SigIgn"] 0
+                verify_log_message 0 "*stacktraces-logging-handling-thread*" 0
```

Comment:
maybe we can add some additional check to see a stack trace coming from one of the other threads? e.g. we know the BIO one is for sure sleeping, right?

Reply:
I'm not familiar wit redis BG threads jobs
i can add a check that `bioProcessBackgroundJobs` is printed 3 times (is it always 3?)

Reply:
currently we have 3 (always), i'd be ok if we check that we check that we have at least one (that way we can see that we extract a trace not only from the main thread).
i guess there's no harm to match it to 3 though.. if we'll ever change the number, the test will fail and we can update it.